rbkit-lib/model/cpustorage.cpp

```diff
@@ -109,25 +132,53 @@ void RBKit::CpuStorage::updateExistingMethod(QMap<int, QVariant> data) {
                 data[RBKit::CeLine].toInt(),
                 data[RBKit::CeSingletonMethod].toInt());
     }
+
+    //increment the method's total count
+    this->nodes[methodName]->incrementTotalCount();
+    qDebug() << "--------updating total count---------";
 }
 
-void RBKit::CpuStorage::traverseCallGraph(RBKit::CpuNodePtr startingNode, int indent)
+void RBKit::CpuStorage::traverseCallGraph(RBKit::CpuNodePtr startingNode,
+                                          QStandardItem &cgRootNode,
+                                          QList<QStandardItem *> *parent=NULL)
```

Comment:
i get an error that `no viable conversion from long to QList<QStandardItem *>`
you see the default argument `=NULL` is the reason i think